app/jobs/document_proofing_job.rb

```diff
@@ -144,7 +153,7 @@ def decrypt_from_s3(timer:, name:, url:, iv:, key:)
       end
     end
     timer.time("decrypt.#{name}") do
-      encryption_helper.decrypt(data: encrypted_image, iv: iv, key: key)
+      normalize_image_file(encryption_helper.decrypt(data: encrypted_image, iv: iv, key: key))
     end
```

Comment:
should we break out decoding as separate from decryption for benchmarking purposes? At the end of the day may be insignificant :shrug:

```
    decrypted = timer.time("decrypt.#{name}") do
      encryption_helper.decrypt(data: encrypted_image, iv: iv, key: key)
    end
    timer.time("decode.#{name}") do
      normalize_image_file(decrypted)
    end
```

Reply:
What do we use the timers for? I don't have a strong opinion either wayy

Reply:
I made the change, figured its an easy one to increase clarity